src/intelligence_layer/core/model.py

```diff
@@ -601,12 +681,31 @@ def to_instruct_prompt(
         instruction: str,
         input: Optional[str] = None,
         response_prefix: Optional[str] = None,
+        instruction_controls: Optional[Sequence[TextControl]] = None,
+        input_controls: Optional[Sequence[TextControl]] = None,
     ) -> RichPrompt:
+        """Method to use a Chat Model like an Instruct Model`.
+
+        Allows the implementation of a custom prompt format for the specific model in use.
```

Comment:
i thought you wanted to remove this one?

Reply:
Oh good catch! Guess it slipped back in during rebasing. Removed it again